index.html

```diff
@@ -1,37 +1,192 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
     <title>My Portfolio</title>
 </head>
-<body> 
- <header class="navmenu">
-    <section class="logo">My logo</section>
-    <nav class="menu">
-    <section class="button">
-        <img class="background" src="Union.png" alt="hamburger">
+
+<body>
+    <header class="navmenu">
+        <h2 class="logo">My logo</h2>
+        <nav class="menu">
```

Comment:
- [ ] Please remove the `padding`/`margin` on the `body` that is creating some whitespace on the sides of the design. 

![image](https://user-images.githubusercontent.com/18730106/153676764-dfff28eb-9a80-4346-a5b2-bfe03c312426.png)